index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
```

Comment:
- [ ] Kindly change this to another `hx` tag like `h2`. A web page should have only one `h1` which is the main header of the page 👍